src/platform_impl/mod.rs

```diff
@@ -18,9 +18,12 @@ mod platform;
 #[cfg(target_os = "emscripten")]
```

Comment:
Question: Will emscripten conflict with websys with some targets?

Reply:
i don't believe so, the target triple used for websys is `wasm32-unknown-unknown`.  i would assume that if someone is building for `wasm32-unknown-emscripten` they wouldn't also have the `websys` feature set.  i could add something like `target_os = "unknown"` to the cfg condition below (line21) to be safe